source/module_base/test_parallel/parallel_global_test.cpp

```diff
@@ -74,6 +77,63 @@ TEST(ParaGlobal,SplitDiag)
 	//std::cout<<GlobalV::DRANK<<" "<<GlobalV::DSIZE<<std::endl;
 }
 
+TEST(ParaGlobal, MyProd)
+{
+    std::complex<double> in[2] = {std::complex<double>(1.0, 2.0), std::complex<double>(-1, -2)};
+    std::complex<double> inout[2] = {std::complex<double>(2.0, 1.0), std::complex<double>(-2, -1)};
+
+    int len = 2;
+    MPI_Datatype dptr = MPI_DOUBLE_COMPLEX;
+    Parallel_Global::myProd(in, inout, &len, &dptr);
+    EXPECT_EQ(inout[0], std::complex<double>(3.0, 3.0));
+    EXPECT_EQ(inout[1], std::complex<double>(-3.0, -3.0));
+}
+
+TEST(ParaGlobal, init_pools)
```

Comment:
```suggestion
TEST(ParaGlobal, InitPools)
```
Please don't use underscores in naming test suite and test names: Googletest suggests against using underscores in naming tests (https://google.github.io/googletest/primer.html).